homework/index.js

```diff
@@ -0,0 +1,22 @@
+// Get the drop down list
+
+const dropDown = document.getElementById('dropDown');
+
+// Fetch data
+
+const promise = fetch(
```

Comment:
You're only using this promise once; so actually there's no need to declare it as a variable - you can just immediately call `then` on it.

However, you will need the actual repo data later on (when you select a repo from the dropdown list), so you will need to store that somewhere; along with filling in the dropdown list! Consider moving functionality into small functions which clearly describe what they do.